lib/LaTeXML/Util/Image.pm

```diff
@@ -336,7 +338,7 @@ sub image_graphicx_complex {
   # Get some defaults from the read-in image.
   my ($imagedpi) = image_getvalue($image, 'x-resolution');
   # image_setvalue($image,debug=>'exception');
-  my ($bg) = image_getvalue($image, 'transparent-color');
+  my ($bg) = image_getvalue($image, 'mattecolor');
```

Comment:
What is the story of `mattecolor`? Just out of curiosity.

Reply:
I don't know the story, but it seems that `mattecolor` is the name used by ImageMagick; not sure if `transparent-color` actually ever worked?